cmd/maya-apiserver/app/command/start.go

```diff
@@ -126,6 +129,22 @@ func NewCmdStart() *cobra.Command {
 	return cmd
 }
 
+func performPreInflightChecks() error {
+	return checkForNDMrelatedCRDs()
+}
+
+func checkForNDMrelatedCRDs() error {
+	_, err := bdc.NewKubeClient().List(metav1.ListOptions{})
+	if err != nil {
+		return fmt.Errorf("precheck for bdc failed: %v", err)
```

Comment:
Let's use `error.Errorf` here. We are using `fmt` for string related operation only.

Reply:
there is errors package of maya which doesn't work for wrapping..

Reply:
so, using fmt..

Reply:
ok.. instead of wrap.. will use errors.Errorf itself